Frontend/ui-driver/src/Strings/EN.js

```diff
@@ -849,5 +849,12 @@ const englishStrings = {
   PLEASE_CONSIDER_CALLING_THEM : "They might not be able to read the chats. Please consider calling them",
   JOIN_A_PLAN_TO_START_EARNING : "Join a Plan to Start Earning!",
   GO_ONLINE_PROMPT_SUBSCRIBE : "You are currently offline.\nJoin a plan to take rides",
-  COMPLETE_PAYMENT_TO_CONTINUE : "To continue using Yatri Sathi, please complete your payment"
+  COMPLETE_PAYMENT_TO_CONTINUE : "To continue using Yatri Sathi, please complete your payment",
+  DOWNGRADE_AVAILABLE_ONLY_FOR_AC_VEHICLES : "Downgrade vehicle option is only available for AC vehicles",
+  DOWNGRADING_VEHICLE_WILL_ALLOW_YOU_TO_TAKE_BOTH_1 : "Downgrading your vehicle will allow you to take both",
```

Comment:
Instead of using three different constructors can we maintain just one constructor like this :`Downgrading your vehicle will allow you to take both {1}  rides and {2} rides`. And in this we can replace `{1}` and `{2}` with the required strings

Reply:
you mean to match the string if {1} exists then replace it with some string ?

Reply:
yes

Reply:
not a very optimal way, later we can try doing it by passing Strings with constructor and also moving translations from JS to purs